pipeline-examples/jobs-in-parallel/jobs_in_parallel.groovy

```diff
@@ -7,13 +7,14 @@ def branches = [:]
 //and jenkins will unite them into a single run of the job
 
 for (int i = 0; i < 4; i++) {
+  def _i = i //if we tried to use i below, it would equal 4 in each job execution.
```

Comment:
ping @TheJefe WDYT? At least about the doc part as that repo is also designed to explain things?